docs/issue_template.md

```diff
@@ -1,15 +1,20 @@
-<!--- The problem --->
-1. Replace this text with a clear statement of the problem that needs to be solved including any relevant supporting files.
-- **If its a software abnormality**, explain how it should act, how it does act, and how to reproduce it.
-- **If you are reporting a security related problem,** DO NOT report here. Email [support@daemo.org](mailto:support@daemo.org) instead.
-- **If this is associated with a strategic milestone** add the `strategy` milestone.
-2. Add a title in the field above that concisely summarizes the problem.
-
-## My proposal
-3. If you have a solution to this problem and you will implement it, continue, otherwise remove the **my proposal** section.
-4. Add yourself as an assignee to this proposal.
-5. Replace this text with a clear statement of the solution you propose to implement, including any relevant supporting files. It can strengthen your proposal to include *pros* and *cons* and your thoughts about the *implications* (short and long term) of it being executed.
+Add a proposal title above that concisely summarizes the problem or suggestion. For best results, add the title after adding all the other parts of this proposal so you can be sure that the title reflects the content.
+
+## Problem
+1. Replace this text with a clear statement of the problem that needs to be solved. If feasible, include specific evidence or examples (e.g., screen shots, data) in support of the understanding of the problem. If this is a bug, [report it in the Daemo repository](https://github.com/crowdresearch/daemo/issues/new) instead of here.
+
+
+## Proposal
+2. If you have a solution to this problem and you will implement it, replace this text with a clear statement of the solution you propose to implement, including any relevant supporting files. It can strengthen your proposal to include *pros* and *cons*. If you have no solution, remove this Proposal section.
+3. Add yourself as an assignee to this proposal.
+4. If this is a strategic milestone, add the `strategy` milestone.
+
+### Implications
+5. Replace this text with short-term (and, if relevant, long-term) implications of executing this proposal. If it is a strategy proposl, long-term implications are required.
+
+## Contact
```

Comment:
For now to make sure attributions are clear, let's change Contact heading to **Person or People Who Added the Proposal to GitHub** 

With a asterisk note below 
**The person who added the proposal to GitHub--- the name in this section doesn't indicate that the person came up with this idea, unless and until explicitly and clearly mentioned.  

and then names
XYZ